src/org/safs/rest/service/DelegateSafsrestAdapter.groovy

```diff
@@ -28,19 +28,4 @@ class DelegateSafsrestAdapter extends HttpClientPOJOAdapter {
 		adapter
 	}
 
-	@Override
```

Comment:
If you don't mind, let's give the Delegate annotation one more try.  I can look closer into any build problem that may occur.